proofpoint-threat-response/info.json

```diff
@@ -0,0 +1,883 @@
+          "title": "Summary",
+          "type": "text",
+          "description": "Specify the alert summary. This argument will populate the Alert Details field.",
+          "tooltip": "Specify the alert summary. This argument will populate the Alert Details field.",
+          "required": false,
+          "editable": true,
+          "visible": true
+        },
+        {
+          "name": "target",
+          "title": "Target",
+          "type": "json",
+          "description": "Specify the target host information in JSON format : \"{\"target\": {...}}\". To see all relevant JSON fields and for more information, see Proofpoint TRAP documentation under \"JSON Alert Source 2.0\".",
+          "tooltip": "Specify the target host information in JSON format : \"{\"target\": {...}}\". To see all relevant JSON fields and for more information, see Proofpoint TRAP documentation under \"JSON Alert Source 2.0\".",
+          "required": false,
+          "editable": true,
+          "visible": true
+        },
+        {
+          "name": "threat_info",
+          "title": "Threat Info",
+          "type": "json",
+          "description": "Specify the threat information in JSON format: \"{\"threat_info\": {...}}\". To see all relevant JSON fields and for more information, see Proofpoint TRAP documentation under \"JSON Alert Source 2.0\".",
+          "tooltip": "Specify the threat information in JSON format: \"{\"threat_info\": {...}}\". To see all relevant JSON fields and for more information, see Proofpoint TRAP documentation under \"JSON Alert Source 2.0\".",
+          "required": false,
+          "editable": true,
+          "visible": true
+        },
+        {
+          "name": "custom_fields",
+          "title": "Custom Fields",
+          "type": "json",
+          "description": "Specify a JSON object for collecting custom name-value pairs as part of the JSON alert sent to Threat Response, in the format: \"{\"custom_fields\": {..}}\". Although there is no limit to the number of custom fields, Proofpoint recommends keeping it to 10 or fewer fields. To see all relevant JSON fields and for more information, see Proofpoint TRAP documentation under \"JSON Alert Source 2.0\".",
+          "tooltip": "Specify a JSON object for collecting custom name-value pairs as part of the JSON alert sent to Threat Response, in the format: \"{\"custom_fields\": {..}}\". Although there is no limit to the number of custom fields, Proofpoint recommends keeping it to 10 or fewer fields. To see all relevant JSON fields and for more information, see Proofpoint TRAP documentation under \"JSON Alert Source 2.0\".",
+          "required": false,
+          "editable": true,
+          "visible": true
+        }
+      ],
+      "enabled": true
+    },
+    {
+      "operation": "close_incident",
+      "title": "Close Incident",
+      "category": "investigation",
+      "annotation": "close_incident",
+      "description": "Close an incident.",
```

Comment:
Update description: `Close an incident in Proofpoint Threat Response based on the input parameters you have specified.`